Update WebDriver recommendations for Selenium 3 to 4 migration (#1685)
MicrosoftDocs/edge-developer: microsoft-edge/webdriver-chromium/index.md

### Using Selenium 4

Selenium WebDriver is an open-source testing framework that can be used on any platform, and provides language bindings for Java, Python, C#, Ruby, and JavaScript.

The Microsoft Edge team recommends [Selenium 4](https://www.nuget.org/packages/Selenium.WebDriver), because Selenium 4 has built-in support for Microsoft Edge (Chromium).  To install Selenium 4, see [Installing Selenium libraries](https://www.selenium.dev/documentation/en/selenium_installation/installing_selenium_libraries).

### Upgrading from Selenium 3

The Microsoft Edge team recommends upgrading existing Selenium 3 tests to Selenium 4, because the Selenium project no longer maintains Selenium 3.  To learn more about upgrading to Selenium 4, navigate to [Upgrade to Selenium 4](https://www.selenium.dev/documentation/webdriver/getting_started/upgrade_to_selenium_4/).

If you're using [Selenium Tools for Microsoft Edge](https://github.com/microsoft/edge-selenium-tools) to add Microsoft Edge (Chromium) support to your Selenium 3 browser tests, update your tests as follows:

> 1. Remove [Selenium Tools for Microsoft Edge](https://github.com/microsoft/edge-selenium-tools) from your project.  It isn't necessary to use Selenium Tools for Microsoft Edge with Selenium 4, because Selenium 4 already has built-in support for Microsoft Edge.

Reviewer: ```suggestion
1. Remove [Selenium Tools for Microsoft Edge](https://github.com/microsoft/edge-selenium-tools) from your project.  You don't need to use Selenium Tools for Microsoft Edge with Selenium 4, because Selenium 4 already has built-in support for Microsoft Edge (Chromium).
```

1. Update your tests to use the built-in `EdgeDriver` and related classes that Selenium 4 provides instead.

> 1. Remove all usages of the `EdgeOptions.UseChromium` property.  This property no longer exists in Selenium 4, because Selenium 4 supports Chromium-based Microsoft Edge only.

Reviewer: ```suggestion
1. Remove all usages of the `EdgeOptions.UseChromium` property.  This property no longer exists in Selenium 4, because Selenium 4 already supports Microsoft Edge (Chromium).
```

Author: Accepted but kept the word "only" because this should make it more clear that Chromium isn't just the default but is the only supported version.

<!-- ====================================================================== -->
## Opt out of diagnostic data collection

By default, Microsoft Edge Driver sends diagnostic data such as the status of the [New Session WebDriver command](https://www.w3.org/TR/webdriver2/#new-session) to Microsoft.  To turn off diagnostic data collection for Microsoft Edge Driver, set the `MSEDGEDRIVER_TELEMETRY_OPTOUT` environment variable to `1`.  For more information about the data that Microsoft Edge Driver collects, see the [Microsoft Edge Privacy Whitepaper](/microsoft-edge/privacy-whitepaper#microsoft-edge-driver).

<!-- ====================================================================== -->

> ## Known issues

Reviewer 2: ```suggestion
## Troubleshooting
```

> These are known issues with using WebDriver to automate Microsoft Edge.

Reviewer 2: ```suggestion
These are known issues or troubleshooting considerations when using WebDriver to automate Microsoft Edge.
```
The 2nd entry isn't a bug, but a common mistake or problem/symptom that people run into, & the fix.
